src/scss/_osu_layout_builder.scss

```diff
@@ -12,3 +12,8 @@
 #drupal-modal .form-item {
   width: initial !important;
 }
+
+.input-icon.fade-zoom-in {
```

Comment:
Need to make your specificity higher, suggest using the id of #drupal-off-canvas as ID's have higher specificity.

Got it working with something like this: #drupal-off-canvas span.input-icon.fade-zoom-in

Reply:
Okay, I see that ID up in the DOM. Definitely more specific.